packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -181,6 +212,11 @@ class _ReorderableListContent extends StatefulWidget {
     @required this.padding,
     @required this.onReorder,
     @required this.reverse,
+<<<<<<< HEAD
+    @required this.physics
+=======
```

Comment:
Looks like some artifacts from rebasing remained :)

Reply:
Apologies- should be fixed.